pkg/operator/k8sutil/tolerations.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package k8sutil
+
+import (
+	"encoding/json"
+
+	"github.com/ghodss/yaml"
+	v1 "k8s.io/api/core/v1"
+)
+
+// This function takes raw YAML string and converts it to Kubernetes Tolerations array
+func YamlToTolerations(raw string, tolerations *[]v1.Toleration) error {
```

Comment:
rather than taking a pointer to the array, I'd suggest the more common pattern is to return the tolerations. For example:
```
func YamlToTolerations(raw string) (tolerations *[]v1.Toleration, error) 
```

Reply:
Thanks for your suggestion. I'm still learning go. I don't remember now why I took this approach before. Fixed.